libs/ledger-live-common/src/families/ethereum/synchronisation.ts

```diff
@@ -167,17 +199,22 @@ export const getAccountShape: GetAccountShape = async (
   tokenAccounts = await prepareTokenAccounts(currency, tokenAccounts, address);
   tokenAccounts = await loadERC20Balances(tokenAccounts, address, api);
   tokenAccounts = await digestTokenAccounts(currency, tokenAccounts, address);
+  log("ethereum", "tokenAccounts", tokenAccounts);
   const subAccounts = reconciliateSubAccounts(tokenAccounts, initialAccount);
+  log("ethereum", "subAccounts", subAccounts);
```

Comment:
Same here for the transactions of those accounts. 
Also log only one of those as tokenAccounts and subAccounts are identitcal AFAIK in the Ethereum family.